diagrams/c4/1_systemContext.puml

```diff
@@ -0,0 +1,23 @@
+@startuml
+!include https://raw.githubusercontent.com/plantuml-stdlib/C4-PlantUML/master/C4_Container.puml
+
+Person(roleUser, "User")
```

Comment:
Is it important to declare `User` as an external person? Just to separate him from `Administrator`.

Reply:
I think - yes, it allows us to clarify existing roles.